src/mon/PGMonitor.cc

```diff
@@ -535,6 +578,10 @@ bool PGMonitor::preprocess_query(MonOpRequestRef op)
 
 bool PGMonitor::prepare_update(MonOpRequestRef op)
 {
+  if (mon->osdmon()->osdmap.require_osd_release >= CEPH_RELEASE_LUMINOUS) {
+    return false;
```

Comment:
I meant the call sites in the dispatch event loop. :)

And speaking of those call sites, see below about disabling it.